pycbc/mchirp_area.py

```diff
@@ -64,16 +65,30 @@ def insert_args(parser):
 
 
 def from_cli(args):
-    return {'mass_limits': {'max_m1': args.src_class_mass_range[1],
-                            'min_m2': args.src_class_mass_range[0]},
-            'mass_bdary': {'ns_max': args.src_class_mass_gap[0],
-                           'gap_max': args.src_class_mass_gap[1]},
+    if args.src_class_mass_gap_max:
```

Comment:
I think you want to put in a check that max gap is not lower than max ns here, otherwise weird things might happen.

Reply:
should it be a warning, a full error or just a check that max_gap > max_ns and in case is it not use max_gap = max_ns?

Reply:
Tito always suggests a parser error in these cases .. see eg https://github.com/gwastro/pycbc/blob/master/bin/pycbc_live#L779

Reply:
thanks!